sdk/core/ts-http-runtime/package.json

```diff
@@ -62,8 +62,8 @@
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "integration-test:browser": "echo skipped",
     "integration-test:node": "echo skipped",
-    "lint": "eslint package.json api-extractor.json src test",
-    "lint:fix": "eslint package.json api-extractor.json src test --fix --fix-type [problem,suggestion]",
+    "lint": "tsx scripts/azure-diff.ts && eslint package.json api-extractor.json src test",
```

Comment:
I recommend introducing this "softly" in a way that does not fail CI quite yet. It's easy to make that change later so feel free to ignore, but I would first introduce it in a way that does not fail the `lint` task and _then_ when things look safe (you checked a bunch of builds, you see what you expect, and nothing that you don't) you can make this fail the `lint` task.